doc/lua_api.txt

```diff
@@ -6962,11 +6966,15 @@ object you are working with still exists.
             (default: `true`)
         * `texture`: A regular texture for the moon. Setting to `""`
             will re-enable the mesh moon. (default: `"moon.png"`, if it exists)
-            Note: Relative to the sun, the moon texture is rotated by 180°.
+            At sunrise, the texture appears non-rotated on the screen if one
+            looks at it.
```

Comment:
Same proposed wording here.

The texture appears non-rotated at sunrise / moonset and rotated 180 degrees (upside down) at sunset / moonrise.